build.sbt

```
@@ -42,7 +42,7 @@ lazy val akkaDependencies = Seq(
   "com.typesafe.akka" %% "akka-http-testkit" % akkaHttpVersion % Test
 )
 
-lazy val awsJavaSdkVersion = "2.16.104"
+lazy val awsJavaSdkVersion = "2.17.14"
```

Making a test change to see if this will make the github actions start
```suggestion
lazy val awsJavaSdkVersion = "2.17.14" 
```

---

@JannikArndt, could you please add @m-dobler to the `moia-oss` organization?

---

Done 👍